backends/bmv2/common/extern.cpp

```diff
@@ -252,4 +252,39 @@ ExternConverter::convertHashAlgorithm(cstring algorithm) {
     return result;
 }
 
+ExternConverter_assert ExternConverter_assert::singleton;
+ExternConverter_assume ExternConverter_assume::singleton;
+
+Util::IJson*
+ExternConverter::convertAssertAssume(ConversionContext* ctxt,
```

Comment:
What will bmv2 do with this call? Is there a corresponding primitive implemented?

Reply:
Yes, I have made a new PR in behavioral-model for adding assert/assume primitives in the simulation.

Reply:
Are these changes already merged?

Reply:
I think they are not, I just made this as a separate PR from previous one, because this one does not include support for assert/assume in ebpf backend yet.

Reply:
For reference, the corresponding changes to bmv2 are in this PR: https://github.com/p4lang/behavioral-model/pull/762

Antonin has suggested some modifications to that PR before he would be willing to accept it.